ThumbnailSequence.cpp

```diff
@@ -1560,7 +1560,7 @@ ThumbnailSequence::CompositeItem::CompositeItem(
 bool
 ThumbnailSequence::CompositeItem::incompleteThumbnail() const
 {
-	return dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
+	return m_pThumb == 0 && dynamic_cast<IncompleteThumbnail*>(m_pThumb) != 0;
```

Comment:
I have no idea why one version works while another doesn't. That being said, your fix shouldn't make any difference really. You will have to dig deeper to find the real reason for the crash. I can't help you there, as it works fine both on Windows and on a virtual machine with a recent version of Linux.